docs/DeveloperGuide.md

```diff
@@ -654,8 +654,51 @@ Names in should only contain alphanumeric characters
 Example: listPerson Alex David
 ```
 
-### Improved Space Sensitivity in Names
+### Stronger Email Input Validation
+- **Background**: Currently, the `Email` of our `Person` is currently validated
+with the format of `local-part@domain`. The local-part and domain have their own
+restrictions with the aim of allowing SnS to reject invalid emails.
+
+- **Issue**: This validation does not strictly comply with the IETF standards for email
+addresses, resulting in invalid emails to be considered as valid by our application.
+For example, `a@12.34` has an invalid domain that does not comply with IETF standards
+but will be considered valid in our application.
+
+- **Enhancement**: We plan on ensuring that `Email` uses stronger validation to comply
+with IETF standards and the following (non-exhaustive) list of RFCs:
+[RFC3696](https://datatracker.ietf.org/doc/html/rfc3696),
+[RFC5322](https://datatracker.ietf.org/doc/html/rfc5322) and
+[RFC6854](https://datatracker.ietf.org/doc/html/rfc6854). As such, SnS will be able
+to more accurately detect invalid email addresses.
+
+### Stronger Telegram Handle Input Validation
+- **Background**: Currently, the `TelegramHandle` of our `Person` is currently validated.
```

Comment:
Currently is repeated (maybe delete one of the 'currently's)

Reply:
Changed